distribution/kernel/carbon-home/repository/resources/conf/templates/repository/conf/tomcat/catalina-server.xml.j2

```diff
@@ -66,7 +66,13 @@
                 <Valve className="org.wso2.carbon.tomcat.ext.valves.RequestCorrelationIdValve"
                        headerToCorrelationIdMapping="{'activityid':'Correlation-ID'}" queryToCorrelationIdMapping="{'RelayState':'Correlation-ID'}"/>
                 <Valve className="org.wso2.carbon.tomcat.ext.valves.CarbonContextCreatorValve"/>
+                {% if http_access_log.useLogger is sameas true %}
```

Comment:
would it be better if we add this to default.json? WDYT @pulasthi7

Reply:
[http_access_log]
useLogger = false

Reply:
@Buddhimah It is not need to add this to default.json as the default is always taken as false when not explicitly defined as "true"